fastlane/Fastfile

```diff
@@ -12,6 +12,7 @@ platform :ios do
       workspace: './Example/BitmovinConvivaAnalytics.xcworkspace',
       code_coverage: true,
       xcodebuild_formatter: 'xcbeautify --renderer github-actions',
+      output_types: '',
```

Comment:
This is an extra change to fix the warning here: https://github.com/bitmovin/bitmovin-player-ios-analytics-conviva/actions/runs/9743153941/job/26886549740#step:7:726